src/models/converters/transactions/details.rs

```diff
@@ -18,20 +18,23 @@ impl MultisigTransaction {
         rejections: Option<Vec<String>>,
         info_provider: &(impl InfoProvider + Sync),
     ) -> ApiResult<TransactionDetails> {
-        let safe_info = info_provider.safe_info(&self.safe.to_string()).await?;
+        let safe_transaction = &self.safe_transaction;
```

Comment:
Why not inline this? Not that I would be against it if it's just for readability, just curious :)